saagieapi/projects/saagie_api.py

```diff
@@ -1251,7 +1258,7 @@ def edit_pipeline(self, pipeline_id, name=None, description=None, emails=None,
             else:
                 raise RuntimeError("Please specify a correct timezone")
 
-        elif is_scheduled == False:
+        elif not is_scheduled:
```

Comment:
We have to specified three case for is_scheduled, when it is set to True, False and None. If we use not is_scheduled, it will take in account when is_scheduled is None and when is_scheduled is False.
None is used when the user want to use the previous scheduler value.
False is used when the user doesn't want to schedule the job
True is used when the user want to add a new schedule for the job

Reply:
👍  I reverted my change